paymentsheet/src/main/java/com/stripe/android/customersheet/analytics/CustomerSheetEvent.kt

```diff
@@ -28,7 +30,8 @@ internal sealed class CustomerSheetEvent : AnalyticsEvent {
                     FIELD_BILLING_DETAILS_COLLECTION_CONFIGURATION to (
                         configuration.billingDetailsCollectionConfiguration.toAnalyticsMap()
                         ),
-                    FIELD_PREFERRED_NETWORKS to configuration.preferredNetworks.toAnalyticsValue()
+                    FIELD_PREFERRED_NETWORKS to configuration.preferredNetworks.toAnalyticsValue(),
+                    FIELD_CUSTOMER_ACCESS_PROVIDER to integrationType.analyticsValue,
```

Comment:
Figured in analytics it would be okay to represent this in `config` rather than have a separate field for it.